python/samples/learn_resources/planner.py

```diff
@@ -4,7 +4,7 @@
 
 from samples.learn_resources.sk_service_configurator import add_service
 from semantic_kernel import Kernel
-from semantic_kernel.planners.sequential_planner import SequentialPlanner
+from semantic_kernel.planners.function_calling_stepwise_planner import FunctionCallingStepwisePlanner
```

Comment:
Would it make sense to maintain a sample for `SequentialPlanner` and one for `FunctionCallingStepwisePlanner`?

Reply:
I can look at adding a sample for the Sequential Planner in the future.